poc/idpf_poplar.sage

```diff
@@ -137,8 +158,18 @@ class IdpfPoplar(Idpf):
 
         next_ctrl = t[bit]
         (next_seed, y) = IdpfPoplar.convert(level, s[bit])
-        if next_ctrl == Field2(1):
```

Comment:
We should make similar bit-to-mask changes for the "if prev_ctrl..." block above.

Reply:
Done, good catch.